tests/e2e/test_cli.py

```diff
@@ -10,6 +10,9 @@
 from autoimport.entrypoints.cli import cli
 from autoimport.version import __version__
 
+# When invoking the CLI for tests, ignore the config outside the repository.
+COMMON_OPTIONS = ["--no-global-config"]
```

Comment:
Instead of creating the `--no-global-config` just for the tests (please correct me if there is any other use case of that flag), I think it would be better to set the environment variable pretty much like how `pydantic` config object works. Let me ask this too to the developer of `maison`

Reply:
I just realized that this could be implemented at `click` level as [they support loading values from environment variables](https://click.palletsprojects.com/en/7.x/options/#values-from-environment-variables) So in the `runner` fixture we could set the value